### ISSUE TITLE
Unable to install homeassistant: No space left on device

### ISSUE THREAD

**Issue by Niorix** [openlumi/homeassistant_on_openwrt#51]
Добрый день!
Установка производится на полностью чистый, только что прошитый шлюз. Ни какаих дополнительных пакетов после прошивки не ставилось.
Лог установки во вложении.
[install.log](https://github.com/user-attachments/files/16739476/install.log)

**Comment by rradar** [comment]
Maybe you don't have space left on your device?

**Comment by Niorix** [comment]
> Maybe you don't have space left on your device?

This is true, but it is assumed that the script can install HA without external storage. It may be possible to change the installation script

**Comment by garg-art2002** [comment]
256 MB is likely min.  I will try with something having 1GB.

**Comment by garg-art2002** [comment]
I just installed it on openwrt 23.5 and used the automated script.  After the done was displayed, telling me that the installation was complete, it took additional 301 MB of space. I think its fair to assume that 500 MB should be made a minimum.

I could not find luci zigbee tools in my System > software > update

**Comment by devbis** [comment]
The script was tuned to install python packages one by one. It should fix the problem of no space left for the script
